### DISCUSSION TITLE
Struggling to use --glob to filter in .ripgreprc

### DISCUSSION THREAD

**Discussion by IamSmeagol** [BurntSushi/ripgrep#2910] [Q&A] [answered]
I just started using ripgrep a few days ago and I can't seem to configure it to ignore one specific directory and it's contents. 

I want to have my `~/Downloads/lsps/lua-lang` folder ignored in every command

Running `rg something --glob '!Downloads/lsps/lua-lang'` works as expected. Then I made a `.ripgreprc` file which only contains
```
--glob 
!Downloads/lsps/lua-lang
```
I also tried
`--glob=!~/Downloads/lsps/lua-lang` 
After running `rg something` the Download/lsps/lua-lang 's contents still appear. Could I get some help please?

I am using  Ubuntu 22.04.5 LTS with WSL 
ripgrep 13.0.0
-SIMD -AVX (compiled)
+SIMD +AVX (runtime)

**Comment by BurntSushi** [comment, marked as answer]
Have you tried running with `--debug`?

Also, the paths you show aren't consistent. Sometimes they are absolute and sometimes they are relative. Relative paths will be interpreted relative to where you're running ripgrep, and you don't say that either.

Also, it would probably work better if you used a `.rgignore` file instead.

Finally, your version of ripgrep is quite old. I only support the latest version of ripgrep.

**Comment by IamSmeagol** [comment]
I got it to work with a `.rgignore` file. Thank you!
